Xamarin.Forms.Core/Cells/CellExtensions.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using Xamarin.Forms.Internals;
+
+namespace Xamarin.Forms
```

Comment:
I had them in Internals first until I realized that there would be a circular dependency, since this class uses classes/interfaces from the Xamarin.Forms namespace.